doc/rst/source/project.rst

```diff
@@ -97,6 +97,19 @@ Calculations of specific great-circle and geodesic distances or for
 back-azimuths or azimuths are better done using :doc:`mapproject` as
 :doc:`project` is strictly spherical.
 
+
+.. figure:: /_images/project_setup.*
+   :width: 500 px
+   :align: center
+
+   Explanation of the coordinate system utilized by project.  The input point
+   (red circle) is given in the original *x-y* (or *lon-lat*) coordinate system and is projected to
+   the *p-q* coordinate system, defined by the center (**C**) and either the end-point
+   (**E**) or azimuth (:math:`\alpha`), or for geographic data a rotation pole **T** (not shown).
+   The blue point has projected coordinates (p,q=0), which is (r,s) in the original
+   coordinate system.  Options **-L** (limit range of *p*) and **-W** (limit range of *q*)
+   can be used to exclude data outside the specified limits (light gray area).
+
```

Comment:
yes and no, I guess.  It is a 40+ line ugly script that spends most of the time doing annotations and dashed lines, etc, and only one line calling project.  It gets long because I wanted to show very clearly what the various items are, and that is a bit tedious.  I may be able to shorten it by a few lines if I spend some more time thinking about it, but it ain't pretty.  Sure this is a good example of GMT usage?

Reply:
I made it  just a bit shorter by removing unnecessary 1x1 subplot and avoid variable duplication.  Still 39 lines.

Reply:
OK, we can leave the toggle button off but eventually should add a tutorial that explains this all step by step since even the figure is a lot to parse. One question - the caption says that the blue point is (p,q=0), but to me it looks like q=0 and p>0.

Reply:
Maybe it should just say (p,0); I do not mean to imply that p is zero.